assets/app/view/game/buy_trains.rb

```diff
@@ -243,6 +237,31 @@ def other_trains(other_corp_trains)
         trains_to_buy
       end
 
+      def price_range(train)
+        if face_value?(train.owner) || face_value?(@corporation)
+          {
+            type: 'number',
+            min: train.price,
+            max: train.price,
+            value: train.price,
+            size: 1,
+          }
+        else
+          {
+            type: 'number',
+            min: 1,
+            max: @corporation.cash,
+            value: 1,
+            size: @corporation.cash.to_s.size,
+          }
+        end
+      end
+
+      def face_value?(entity)
```

Comment:
copy and pasted, can you call the step method?

Reply:
Ok